### PR TITLE
Refactored get_token to a utils module

### LINKED ISSUE #8: Refactor get_token to a util so it's reusable outside of OpenDiscussionsApi
We'll need this for generating user sessions.

### PR DESCRIPTION
**Pull request by rhysyngsun**
#### What are the relevant tickets?
Fixes #8 

#### What's this PR do?
Refactors get_token to a util

#### How should this be manually tested?
Tests should pass

### CODE REVIEW

File: open_discussions_api/client.py [hunk 1]
```diff
@@ -1,12 +1,8 @@
 """open-discussions api client"""
-import time
-
-import jwt
 import requests
 
 from .users.client import UsersApi
-
-EXPIRATION_DELTA_SECONDS = 60 * 60
+from .utils import get_token
```
Comment (giocalitri) [inline open_discussions_api/client.py]: this my first review in this project and I do not know if we used different conventions, but I thing we should avoid to use relative imports
But please disregard if the convention is that it is OK to use them.

Comment (rhysyngsun) [reply open_discussions_api/client.py]: We use them in `edx-api-client`, which I based this project off of, so the style carried over unintentionally. I'll fix it.

Comment (rhysyngsun) [reply open_discussions_api/client.py]: Actually, even worse, this project was inconsistent. Fixed now.

**Comment by rhysyngsun** [comment]
@giocalitri feedback addressed

**Comment by giocalitri** [comment]
👍

**Pull request merged**